backend/src/main/java/gov/cdc/usds/simplereport/config/SecurityConfiguration.java

```diff
@@ -37,41 +37,36 @@
 @ConditionalOnWebApplication
 public class SecurityConfiguration extends WebSecurityConfigurerAdapter {
```

Comment:
There don't appear to be any substantive changes to this file. It's a little hard to tell, because of all the whitespace and re-wrapping. Please don't reformat code as part of substantive PRs—it makes it harder to see the changes and harder to understand the commit history.